plasmapy/simulation/particle_tracker.py

```diff
@@ -725,15 +755,17 @@ def run(
             of `~plasmapy.simulation.particle_tracker.ParticleTracker`. See `~plasmapy.simulation.particle_tracker.AbstractStopCondition`
             for more details.
 
-        save_routine: `~plasmapy.simulation.particle_tracker.AbstractSaveRoutine`
+        save_routine : `~plasmapy.simulation.particle_tracker.AbstractSaveRoutine`
             An instance of `~plasmapy.simulation.particle_tracker.AbstractSaveRoutine` which determines which
             time steps of the simulation to save. See `~plasmapy.simulation.particle_tracker.AbstractSaveRoutine` for more details.
 
         dt : `~astropy.units.Quantity`, optional
             An explicitly set time step in units convertible to seconds.
             Setting this optional keyword overrules the adaptive time step
-            capability and forces the use of this time step throughout. If a tuple
-            of time steps is provided, the adaptive time step will be clamped
+            capability and forces the use of this time step throughout.
+
+        dt_range : tuple of shape `(2,)` of `~astropy.units.Quantity`, optional
```

Comment:
Looks good - what happens if the user tries to set both? That should raise an exception in the _validate_run_inputs method.